src/lib/__tests__/RoutingManager-test.js

```diff
@@ -545,4 +547,54 @@ describe('RoutingManager', () => {
       });
     });
   });
+
+  describe('windowTitle', () => {
+    let setWindowTitle = jest.fn();
+
+    Object.defineProperty(window.document, 'title', {
+      get() {
+        return '';
+      },
+      set(value) {
+        setWindowTitle(value);
+      },
+    });
```

Comment:
There is an easier way to mock getters/setters
https://jestjs.io/docs/en/jest-object.html#jestspyonobject-methodname-accesstype

The nice thing about this way, is that you can later restore the mock, which you don't at the moment.

Reply:
That's great, I didn't think of this usage. Thanks!